### PR TITLE
Added synology support

### PR DESCRIPTION
**Pull request by CMBoii**
This PR is aimed to fix the compatibility issues of wg-easy with Synology machines. 

The only change needed is present in the Wireguard.js file.

### CODE REVIEW

File: src/lib/WireGuard.js [hunk 1]
```diff
@@ -84,7 +84,9 @@ module.exports = class WireGuard {
 [Interface]
 PrivateKey = ${config.server.privateKey}
 Address = ${config.server.address}/24
-ListenPort = 51820`;
+ListenPort = 51820
+PostUp = iptables -A FORWARD -i %i -j ACCEPT; iptables -A FORWARD -o %i -j ACCEPT; iptables -t nat -A POSTROUTING -o ovs_eth0 -j MASQUERADE
+PostDown = iptables -D FORWARD -i %i -j ACCEPT; iptables -D FORWARD -o %i -j ACCEPT; iptables -t nat -D POSTROUTING -o ovs_eth0 -j MASQUERADE`;
```
Comment (snowzach) [inline src/lib/WireGuard.js:89]: This change would effect everyone, not everyone will have interface ovs_eth0

**Comment by SeanChangX** [comment]
I have the same problem of running it on synology nas. I'm wandering what can I do to let it works on the latest version?

**Comment by WeeJeWel** [comment]
Interestingly, wg-easy runs fine on my DSM7 without this PR?

**Comment by WeeJeWel** [comment]
Please create a Wiki article (In English!!)

**Pull request closed without merging**